userguide/system.rst

```diff
@@ -729,15 +729,37 @@ Self-Encrypting Drives
 
 %brand% version 11.1-U5 introduced Self-Encrypting Drive (SED) support.
 
-Three types of SED are supported:
+These SED specifications are supported:
 
 * Legacy interface for older ATA devices. **Not recommended for
   security-critical environments**
 
-* TCG OPAL 2 standard for newer consumer-grade devices (HDD or SSD over
-  PCIe or SATA)
+* `TCG Opal 1 <https://trustedcomputinggroup.org/wp-content/uploads/Opal_SSC_1.00_rev3.00-Final.pdf>`_
+  legacy specification
 
-* TCG Enterprise standard for newer enterprise-grade SAS devices
+* `TCG OPAL 2 <https://trustedcomputinggroup.org/wp-content/uploads/TCG_Storage-Opal_SSC_v2.01_rev1.00.pdf>`__
+  standard for newer consumer-grade devices
+
+* `TCG Opalite <https://trustedcomputinggroup.org/wp-content/uploads/TCG_Storage-Opalite_SSC_FAQ.pdf>`__
+  is a reduced form of OPAL 2
+
+* `TCG Pyrite 1 <https://trustedcomputinggroup.org/wp-content/uploads/TCG_Storage-Pyrite_SSC_v1.00_r1.00.pdf>`__
+  and
+  `TCG Pyrite 2 <https://trustedcomputinggroup.org/wp-content/uploads/TCG_Storage-Pyrite_SSC_v2.00_r1.00_PUB.pdf>`__
+  provide a logical equivalent of the legacy ATA security for non-ATA
+  devices. Hardware encryption is removed and only the drive firmware is
+  used to protect the device.
+
+  .. warning:: Pyrite Version 1 does not have PSID support and **can
+     become unusable if the password is lost.**
+
+* `TCG Enterprise <https://trustedcomputinggroup.org/wp-content/uploads/TCG_Storage-SSC_Enterprise-v1.01_r1.00.pdf>`__
+  for high performance systems
```

Comment:
I am not sure how this is related to performance.  I'd say more to systems with many data disks, not used for booting, since it does not provide mechanisms to ask for for unlocking password before OS boot.